src/StatusQ/Controls/StatusSlider.qml

```diff
@@ -31,7 +31,7 @@ Slider {
         id: bgRect
 
         x: root.leftPadding
-        y: root.topPadding
+        y: root.topPadding + (root.availableHeight - height) / 2
```

Comment:
How about having pixel alignment?
```suggestion
        y: root.topPadding + ((root.availableHeight - height) / 2).toFixed()
```

Reply:
This is a good point. But should we consider that topPadding is real value?

Reply:
@alexandraB99 when you have free time - check @stefandunca  suggested code - results are very interesting :D

Reply:
> This is a good point. But should we consider that topPadding is real value?

It is a valid concern and we should strive for getting this pixel aligned

Reply:
> This is a good point. But should we consider that topPadding is real value?

I consider that the user of the control has the responsibility to sanitize the values and do it efficiently (only when you know might be needed).

So anything coming from the interface can be assumed that is valid. Same in the provided example, if we know that all values are multiple of two (due to dual spacing or another way) then should not be sanitized. We already had some issues hard to track/fix due to pixel misalignment (blurry emojis, some rare corner cases in redrawing) so taking care to stay aligned should be considered.

> @alexandraB99 when you have free time - check @stefandunca suggested code - results are very interesting :D "Free will" should never be mentioned as it appears when you mention it :)

I had a quick look and the way `toFixed()` generates this side effect of malforming the value is concerning, maybe a Qt JS compiler bug/"optimization". I have never experienced it but plan to follow up later on.